components/resources/brave_news_strings.grdp

```diff
@@ -148,6 +157,9 @@
   <message name="IDS_BRAVE_NEWS_CHANNEL_SCIENCE" desc="Translation of the 'Science' channel">
     Science
   </message>
+  <message name="IDS_BRAVE_NEWS_CHANNEL_SPORT" desc="Translation of the 'Sport' channel">
+    Sport
+  </message>
   <message name="IDS_BRAVE_NEWS_CHANNEL_SPORTS" desc="Translation of the 'Sports' channel">
     Sports
   </message>
```

Comment:
Are these different?

Reply:
Unfortunately. Sport/Sports. The backend channel taxonomy needs to undergo a big cleanup but it's not going to happen for a little while at least.

Until then, there are a bunch of channels which are nearly identical (they've been added a bit haphazardly on the backend).

- Entertainment/Entertainment news/Film & TV/Celebrity News
- Tech news/Tech reviews/Technology
- Top news/Top sources/World news (these might be intentional)
- Sport/Sports

Happy to leave these untranslated (it falls back to the english channel name) but I thought this might be a good stopgap.

Reply:
How diabolical 🙀